pyvista/core/filters/image_data.py

```diff
@@ -950,6 +951,268 @@ def contour_labeled(
         _vtk.vtkLogger.SetStderrVerbosity(verbosity)
         return cast(pyvista.PolyData, wrap(alg.GetOutput()))
 
+    def point_voxels_to_cell_voxels(self, copy: bool = True):
```

Comment:
Possible alternative name: `points_to_cells` to steer away from the confusion regarding the definition of a voxel